testdata/p4_16_samples/psa-register-read-write-bmv2.stf

```diff
@@ -0,0 +1,25 @@
+# cmd | port | dest | src | ethertype
+packet 4 000000000001 000000000000 ffff
+expect 3 000000000003 000000000000 ffff $
+
+packet 4 000000000002 000000000000 ffff
+expect 3 000000000003 000000000000 ffff $
+
+packet 4 000000000003 000000000000 ffff
+expect 3 000000000003 000000000000 ffff $
+
+packet 0 000000000003 000000000000 ffff
+expect 3 000000000003 000000000000 ffff
+
+register_read cIngress.regfile 1
+# expect 3
```

Comment:
should be `check_register cIngress.regfile(1) = 3`?

Reply:
Thanks! I use `register_read` because in the help menu of psa_switch_CLI (behavioral-model/targets/psa_switch), there is a `register_read` RuntimeCmd. I modified STF a little so that it can take in this RuntimeCmd for register.

Reply:
Hi Chris, sorry I might have misunderstood you earlier. I think I now understand: you mean we need to automate the check of the result of the register_read command (like assert in c). I don't think STF has implemented check_register yet. But I think it would be nice to have I think look into it.